packages/rum/test/benchmarks/profiler.js

```diff
@@ -91,35 +103,46 @@ function gatherRawMetrics(browser, url) {
          * Resolve the promise once we measure the size
          * of the payload to APM Server
          */
+        // await context.close()
```

Comment:
Did you mean to commit this line?

Reply:
Forgot to remove it, it's not required to close it, since closing browser would remove it anyways.